README.md

```diff
@@ -29,7 +25,7 @@ markdown-toc-refresh-toc -->
 
 ## Project Background
 
-Preflight was originally designed to automate Jetstack's production readiness
+Jetstack Secure was originally designed to automate Jetstack's production readiness
 assessments.
 These are consulting sessions in which a Jetstack engineer inspects a customer's
 cluster to suggest improvements and identify configuration issues.
```

Comment:
This copy reflects the original idea of Preflight, but Jetstack Secure encompasses more things. I would use similar terms to what we say in: https://www.jetstack.io/jetstack-secure/.